docs/modules/ROOT/pages/how-to-guides/Import-code/proc_importing_code.adoc

```diff
@@ -214,7 +214,40 @@ For more information about devfile contents requirements, see the "What is outer
 +
 **Solution:** Select a predefined runtime type from the **Review and configure for deployment** view. Make sure you meet all of the {ProductName} <<repository requirements>> for the runtime you select, or configure your own <<custom build and deployment instructions>>.
 
-[role="_additional-resources"]
-.Additional resources
-For information about importing and configuring code to {ProductName} using the CLI, see link:https://redhat-appstudio.github.io/docs.stonesoup.io/Documentation/main/getting-started/getting_started_in_cli/#getting-started-in-the-cli[Getting started in the CLI
-].
\ No newline at end of file
+=== Creating a Red Hat Container Registry token
+
+After importing your code, configure a Red Hat Container Registry token to improve your application-building experience. You can access this token by creating Red Hat registry service accounts. The registry service accounts enable you to retrieve content from registry.redhat.io. Registry.redhat.io helps you manage the automation of your applications. 
```

Comment:
SR--> After importing your code, configure a Red Hat Container Registry token to improve your application-building experience. You can access this token by creating a Red Hat registry service account where you can retrieve content from registry.redhat.io to manage the automation of your applications. 

is _registry.redhat.io_ a web address? if so, make it a link?